src/main/java/jenkins/plugins/collabnet/steps/PublishEventStep.java

```diff
@@ -93,8 +83,8 @@ public StepExecution start(StepContext context) throws Exception {
      *
      * @return the server URL
      */
-    public String getServerUrl() {
-        return serverUrl;
+    public String getWebhookUrl() {
+        return webhookUrl;
     }
 
     public String getCredentialsId() {
```

Comment:
Are credentials needed for the new step?

Reply:
Yeah, credentials are not required at least for now. But if we introduce traceability view for pipeline projects then credentials are required. will comment credentials for now.